assets/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@massds/mayflower-theme",
+  "version": "1.0.0",
+  "description": "Mayflower theming assets, including color variables, fonts, icons and other imagery, along with componentized scss used to style Mayflower markup",
+  "main": "index.js",
+  "scripts": {
+    "test": "stylelint",
+    "stylelint": "stylelint '**/*.scss' --syntax scss --formatter verbose",
+    "stylelint-fix": "stylelint '**/*.scss' --fix --syntax scss --formatter verbose"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/massgov/mayflower/tree/master/assets"
+  },
+  "keywords": [
+    "theme",
+    "scss"
+  ],
+  "author": "Massachusetts Digital Services (MDS)",
+  "license": "MIT",
+  "dependencies": {
+    "bourbon": "^5.1.0",
+    "bourbon-neat": "^1.7.2",
```

Comment:
Can you add a bit more context for how/what `bourbon` and `bourbon-neat` are used for? This is a really large PR, making it hard to review and grasp all at once. I'm wondering if there are ways to either replace them (if it's minor) or break them out as separate packages so that this core Sass package doesn't have them as dependencies.

Reply:
agreed, this PR should be broken down into several steps. 
I added style lint to scss (something we have been wanting to do) so that the package won't have malformatted scss, and would enforce a check it going forward. which is causing a lot of diffs. I will break that piece out first. 
After chatting with @rbayliss about the issue that I'm having with the paths for external dependencies (bourbon, bourbon-neat), there isn't a straightforward way to resolve it, and the plan is to do an audit and cleanup those dependencies. I will make a separate PR for that.

Reply:
bourbon and bourbon-neat were used in a lot of component scss as mixins and variables... It might take me a while to identify all the cases that we are using those for